### DISCUSSION TITLE
Ability to run cron jobs on specific preview deployments

### DISCUSSION THREAD

**Discussion by Fiaxhs** [vercel/community#2215] [Idea]
Hi 👋 

I know that cron jobs [are ignored on preview deployments](https://github.com/orgs/vercel/discussions/1928), but I think that being able to run them against specific ones would be great. 

We have a sandbox env where we'd like to have our cron jobs running. Currently we use qstash to run them but we'd rather have everything on Vercel.

Do you have any plans to extend this feature?

**Comment by thomas779** [comment]
+1

> Cron jobs are invoked only for [production](https://vercel.com/docs/concepts/deployments/environments#production) deployments. [Preview](https://vercel.com/docs/concepts/deployments/preview-deployments) deployments are ignored.

Why ignore Preview deployments?

**Comment by FNDEVVE** [comment]
I wasted some time trying to debug it because there is not a single mention of cron jobs not working on "preview" deployments. Please either support it as a toggle or mention it.

**Comment by rsimonton** [comment]
https://vercel.com/guides/how-to-setup-cron-jobs-on-vercel#deploy-your-project

> Once your project is deployed on Vercel, our build process will create the cron job. Cron jobs are invoked only for production deployments. Preview deployments are ignored.

That said how are supposed to test cron processes if we can only run them for production deployments?
